### DISCUSSION TITLE
Add UK date format

### DISCUSSION THREAD

**Discussion by adhawkins** [jonaswinkler/paperless-ng#1014] [Feature Requests]
The UK date format is dd/mm/yyyy. It would be useful to have this available as an option, also allowing the selection of 'long' format using English months. 

It can currently be worked around by selecting 'French' as the date format, but this then uses French month names in the 'long' format.

**Comment by jonaswinkler** [comment]
I believe thi is already available?

![image](https://user-images.githubusercontent.com/17569239/118394980-2c2d5100-b648-11eb-9054-737dbe28818d.png)

Or are you looking for something else?

**Comment by adhawkins** [comment]
Hi,

It's not on my version. I'm still on v1.1.3 though. Has it been added since then?

![image](https://user-images.githubusercontent.com/1628607/118414013-26f8f200-b69a-11eb-9328-2fb77a0bf371.png)

Andy

**Reply by jonaswinkler** [reply]
Yes, added in 1.1.4.

https://paperless-ng.readthedocs.io/en/latest/changelog.html#paperless-ng-1-1-4

**Comment by adhawkins** [comment]
Got it, thanks. Will make a point to upgrade!

Apologies for the noise.

Andy

**Reply by adhawkins** [reply]
Ok. Updated and feature is now there. Can close this if you want.
